### PR TITLE
Fixing DiagnosticScript method reference

### PR DESCRIPTION
**Pull request by shashwattrivedi**
Fixed method name reference for TryResolveStoragePrivateUrl
Changed ErrorAction to SilentlyContinue for Get-ItemProperty

### CODE REVIEW

File: ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1 [hunk 1]
```diff
@@ -154,12 +154,12 @@ try
...
                     $error.clear()
                     $privateip = $null
-                    $privateip = Resolve-DnsName $serviceUrl
+                    $privateip = Resolve-DnsName $serviceUrl -ErrorAction SilentlyContinue
 
                     if( $privateip -eq $null)
                     {
-                        TraceMessage "`nFailed to resolve $serviceUrl"  "Red"
-                        $dns.PrivateIp = $error
+                        TraceMessage "`nFailed to resolve1 $serviceUrl"  "Red"
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1:161]: why was this change required, resolve -> resolve1

File: ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1 [hunk 2]
```diff
@@ -317,34 +317,43 @@ catch
...
-if ( ($CurrentDefaultSettings[-3] -eq '8' ) -or ($CurrentDefaultSettings[-3] -eq 'a') )
+
+TraceMessage "`n=============Fetching TLS settings==================="  "Yellow"
+
+try
 {
-    $DefaultSettings = "TLS 1.2;" + $DefaultSettings
-}
+    $CurrentDefaultSettings =  ‘{0:x}‘ -f  (Get-ItemProperty 'HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings' -ErrorAction SilentlyContinue).SecureProtocols
+    $DefaultSettings = ""
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1:326]: Should we trace this settings?

Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1:326]: Report will have this settings

File: ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1 (hunk 1 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1]: fixed

File: ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1 [hunk 3]
```diff
@@ -89,35 +89,36 @@ function TraceMessage([string] $message, [string] $color="Yellow")
...
-    TraceMessage "`nLoaded ADAL library from $ActiveDirectoryDllPath"
+    TraceMessage "Loaded ADAL library from $ActiveDirectoryDllPath"
     Add-Type -Path $DiagnosticScriptsDllPath
-    TraceMessage "`nLoaded $DiagnosticScriptsDllPath"
+    TraceMessage "Loaded $DiagnosticScriptsDllPath"
 }
 catch
 {
-    TraceMessage "`n $($_.Exception.Message)"
-    TraceMessage "`n $($_.Exception.StackTrace)"
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1:99]: Not sure why this was removed, hope this is expected

Comment (kajai987) [reply ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1:99]: the `n
